code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -158,6 +158,11 @@
 				if(!target.is_open_container() && !ismob(target) && !istype(target, /obj/item/weapon/reagent_containers/food) && !istype(target, /obj/item/slime_extract) && !istype(target, /obj/item/clothing/mask/cigarette) && !istype(target, /obj/item/weapon/storage/fancy/cigarettes))
 					user << "\red You cannot directly fill this object."
 					return
+				if(istype(target, /obj/item/clothing/mask/cigarette))

```

Comment:
You can, because 3 lines up from this, cigarettes (as well as cigarette packs, to handle pack lacing) have a special exception to allow injection.

On live, cigarettes spawn full of nicotine, so you can't inject them until you've burned some of it off. With the changes, cigarettes are empty until you remove them from the pack (because that's how pack lacing works for some reason), so this stops you from injecting cigarettes that haven't been properly filled yet.